css/ampdoc.css

```diff
@@ -321,6 +321,10 @@ amp-story .amp-active > div {
   display: none !important;
 }
 
+amp-story .i-amphtml-loader-background {
```

Comment:
Let's add this selector to the pre-existing rule to hide loaders from #24895 and leave a comment with respect to the intention (of hiding amp-loaders from stories) and perhaps also a TODO to remove once loaders are no longer imported for stories.